framework/pipeline/Trigger_all_increamental_query_to_Stage1np.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
The name has a misspelling in it - can you rename this to be Trigger_all_incremental_query_to_Stage1np.json